client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -58,6 +58,10 @@ export default function IssueMultipleLicensesForm( {
 		allProducts?.filter(
 			( { family_slug }: { family_slug: string } ) => family_slug !== 'jetpack-packs'
```

Comment:
Should filter only the specific set of jetpack products that go in the allProducts list. Excluding WooCommerce bookings from this list.

![image](https://user-images.githubusercontent.com/5550190/227391747-ea0e4cc8-95bc-4101-855f-ea4397e1d2e2.png)

My billing scheme only had this single product, but it appears on the Jetpack products list as well.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/74509/commits/d226242d0b2b1a4da5554bc6cd55def0816a6807